pkg/chart/chart_test.go

```diff
@@ -453,55 +455,88 @@ func newHTTPClient(charts []Details, userAgent string) HTTPClient {
 	}
 	index := &repo.IndexFile{APIVersion: "v1", Generated: time.Now(), Entries: entries}
 	return &clientWithDefaultUserAgent{
-		client:    &fakeHTTPClient{repoURLs, chartURLs, index, userAgent},
+		client: &fakeHTTPClient{
+			repoURLs:  repoURLs,
+			chartURLs: chartURLs,
+			index:     index,
+			userAgent: userAgent,
+		},
```

Comment:
Just switched to labeling the composite literal fields to avoid adding `nil` on the end for the `requests` (which aren't required for initialization)

Reply:
I was unaware of that change of behavior, can you add a comment? If you initialize it like this `request` is an empty slice instead?

Reply:
Yep, from https://golang.org/doc/effective_go.html#composite_literals "However, by labeling the elements explicitly as field:value pairs, the initializers can appear in any order, with the missing ones left as their respective zero values."

There's no change of behaviour here - I added the requests slice in this PR (and it should get a nil value on initialization). I just don't like specifying `nil` in an initialization without a label as you then need to scroll to find out what it was being initialized to nil to see if it's relevant, whereas with a labelled initialization, I can assume all the relevant info is there.